Source/WebInspectorUI/UserInterface/Controllers/ConsoleManager.js

```diff
@@ -293,6 +297,17 @@ WI.ConsoleManager = class ConsoleManager extends WI.Object
         WI.objectStores.consoleSnippets.putObject(snippet);
     }
 
+    _collectFailedSourceMapConsoleMessage(message)
+    {
+        if (!WI.settings.experimentalGroupSourceMapErrors.value)
+            return;
+
+        let isNetworkError = message.source === WI.ConsoleMessage.MessageSource.Network && message.level === WI.ConsoleMessage.MessageLevel.Error;
```

Comment:
id turn this into an early return instead so that we dont have to call `WI.networkManager.isSourceMapURL` on every single console message

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.